tests/integration/cypress/web/02.html

```diff
@@ -1,22 +1,23 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <meta charset="utf-8">
+    <meta charset="utf-8" />
     <title>Base</title>
-    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" />
   </head>
   <body>
     <div class="container">
       <div class="row">
         <div class="col-xs-4">
           <h1>Register</h1>
-          <hr><br>
+          <hr />
+          <br />
```

Comment:
I guess all these auto close tags were made by the linter. I’m fine with it but just to let you know that they are not needed